<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Translation</class>
 <widget class="QDialog" name="Translation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Translation</string>
  </property>
  <widget class="QPushButton" name="pBUpdateTsFile">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>350</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;To update .ts file with tranlating click here&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Update .ts File</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>521</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="minimumSize">
       <size>
        <width>170</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Manual translating file save path</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lEManualTranlatingfileSavePath">
      <property name="minimumSize">
       <size>
        <width>170</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Select Directry to save google tranlating input file&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>Select dir</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>370</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>close</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>521</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="minimumSize">
       <size>
        <width>170</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Select Manual Translated file</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lEManualTranlatedFile">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pBSelectGoogleTranlatedFile">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Select Google tranlator tranlated file &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>Select</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>521</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="minimumSize">
       <size>
        <width>170</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Select updating .ts file</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lEupdatingTsFile">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pBselectUpdatingTsFile">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Select ts file that need to update with tranlation&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>Select</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pBGenarateTranlateFile">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>160</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;To genarate Google tranlation input file click here&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Genarate </string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>521</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="minimumSize">
       <size>
        <width>170</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Select Translating .ts file</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lETranlatingTsFilePath">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pBSelectTranalatingFile">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Select qt tranlation ts file that need to translate&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>Select</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>521</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="minimumSize">
       <size>
        <width>170</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Select Google Translated file</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lEGoogleTranlatedFile">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pBSelectGoogleTranlatedFile_2">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Select Google tranlator tranlated file &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>Select</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pBSelectTranalatingFile</sender>
   <signal>clicked()</signal>
   <receiver>Translation</receiver>
   <slot>tranlatingTsFileSelectClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>517</x>
     <y>16</y>
    </hint>
    <hint type="destinationlabel">
     <x>927</x>
     <y>37</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pBSelectGoogleTranlatedFile_2</sender>
   <signal>clicked()</signal>
   <receiver>Translation</receiver>
   <slot>googleTranslatedFileSelectClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>492</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>780</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Translation</receiver>
   <slot>manualTranlatingFileSavePathSelectClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>485</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>817</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pBSelectGoogleTranlatedFile</sender>
   <signal>clicked()</signal>
   <receiver>Translation</receiver>
   <slot>manualTranslatedFileSelectClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>496</x>
     <y>257</y>
    </hint>
    <hint type="destinationlabel">
     <x>748</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pBselectUpdatingTsFile</sender>
   <signal>clicked()</signal>
   <receiver>Translation</receiver>
   <slot>updatingTsFileSelectClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>493</x>
     <y>297</y>
    </hint>
    <hint type="destinationlabel">
     <x>748</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pBUpdateTsFile</sender>
   <signal>clicked()</signal>
   <receiver>Translation</receiver>
   <slot>updateTsFilePushButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>318</x>
     <y>371</y>
    </hint>
    <hint type="destinationlabel">
     <x>674</x>
     <y>355</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>Translation</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>530</x>
     <y>386</y>
    </hint>
    <hint type="destinationlabel">
     <x>772</x>
     <y>389</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pBGenarateTranlateFile</sender>
   <signal>clicked()</signal>
   <receiver>Translation</receiver>
   <slot>genaratePushButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>315</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>758</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>tranlatingTsFileSelectClicked()</slot>
  <slot>googleTranslatedFileSelectClicked()</slot>
  <slot>manualTranlatingFileSavePathSelectClicked()</slot>
  <slot>genaratePushButtonClicked()</slot>
  <slot>manualTranslatedFileSelectClicked()</slot>
  <slot>updatingTsFileSelectClicked()</slot>
  <slot>updateTsFilePushButtonClicked()</slot>
 </slots>
</ui>
